core/transport/src/main/proto/grpc/meta_master.proto

```diff
@@ -237,14 +237,21 @@ enum MetaCommand {
 
 message RegisterMasterPOptions {
   repeated grpc.ConfigProperty configs = 1;
+  optional int64 startTimeMs = 2;
+  optional int64 primacyChangeTimeMs = 3;
```

Comment:
why put it in register master instead of master heartbeat, will prefer in master heartbeat

Reply:
and is it lastLosePrimacyTime ? then where is gain primacy time?

Reply:
> why put it in register master instead of master heartbeat, will prefer in master heartbeat

Because these values are only supposed to change when register master.

> and is it lastLosePrimacyTime ? then where is gain primacy time?
Yes it is actually `lastLosePrimacyTime`.

Because this protocol is for standby master to send information to the primary master,
so the last gain primacy time will be an old value (maybe not so useful), so I decide not to include them.
We can tell which standby master is the last primacy master by looking at `lastLosePrimacyTime`.